refgenconf/refgenconf.py

```diff
@@ -2087,11 +2164,148 @@ def _select_genomes(self, genome=None, strict=False, order=None,
                 else:
                     missing.append(g)
             if missing:
-                _LOGGER.warning("Genomes do not include: {}".format(", ".join(missing)))
+                _LOGGER.warning(
+                    "Genomes do not include: {}".format(", ".join(missing)))
             return None if not filtered else filtered
         return genomes if not all(x in genomes for x in genome) else genome
 
 
+def config_upgrade(target_version, filepath, force=False):
+    """
+    upgrade the config file to a target version,
+    and alter genome_folder stucture
+
+    :param str target_version: the version updated to
```

Comment:
missing `param`

Reply:
5ae7e2c7f0cf50cf57eed265573fa168b1a91a3e

Reply:
this should be resolved now..